Description: Fix DetermintantalRepresentations package test on i386
 Use more forgiving epsilons for the clean function to avoid failures.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1561
Applied-Upstream: 1.17, https://github.com/Macaulay2/M2/commit/0de9e42,
 https://github.com/Macaulay2/M2/commit/e4cb68e
Last-Update: 2020-11-02

--- a/M2/Macaulay2/packages/DeterminantalRepresentations.m2
+++ b/M2/Macaulay2/packages/DeterminantalRepresentations.m2
@@ -1345,13 +1345,13 @@
 SRR=RR[x1,x2,x3]
 fRR=sub(f, SRR)
 reps = detRep fRR
-assert(all(reps, L -> clean(1e-10, fRR - det L) == 0))
+assert(all(reps, L -> clean(1e-9, fRR - det L) == 0))
 SCC=CC[x1,x2,x3]
 fCC=sub(f, SCC)
 reps = detRep fCC
-assert(all(reps, L -> clean(1e-10, fCC - det L) == 0))
+assert(all(reps, L -> clean(1e-9, fCC - det L) == 0))
 reps = detRep(fCC, HyperbolicPt => matrix{{3_RR},{2},{-7}}) -- representations are not monic
-assert(all(reps, L -> clean(1e-8, fCC - det L) == 0))
+assert(all(reps, L -> clean(1e-6, fCC - det L) == 0))
 ///
 
 TEST /// -- Cubic case: homogeneous, 3 variables
